proto/utxorpc/v1alpha/query/query.proto

```diff
@@ -109,15 +109,34 @@ message ReadDataResponse {
   ChainPoint ledger_tip = 2; // The chain point that represent the ledger current position.
 }
 
+// Request to get a transaction by hash
+message GetTxRequest {
```

Comment:
proto lint warnings suggest using names that match the service method. Here we should use `ReadTxRequest`.